apps/condo/next.config.js

```diff
@@ -32,7 +32,7 @@ const apolloGraphQLUrl = `${serverUrl}/admin/api`
 const addressServiceUrl = conf['ADDRESS_SERVICE_URL']
 const mapApiKey = conf['MAP_API_KEY']
 const behaviorRecorder = { 'plerdy': conf['BEHAVIOR_RECORDER_PLERDY_CONFIG'] }
-const featureFlagsConfig = conf['FEATURE_FLAGS_CONFIG']
+const canEnableSubscriptions = conf['CAN_ENABLE_SUBSCRIPTIONS'] === 'true'
```

Comment:
why we need featureflag and env variable together?

Reply:
If you turn this one on, it practically disables the service and prompts u to give us some money :-) I believe if turned on by mistake this will render us a PR nightmare, so I decided to take a safer path and lock it behind .env as well